src/providers/twitch/api/Helix.hpp

```diff
@@ -329,6 +330,16 @@ struct HelixChatSettings {
     }
 };
 
+class HelixUserList {
+    public:
+        std::unordered_set<QString> users;
+        int total;
+
+        HelixUserList();
```

Comment:
The constructor doesn't do anything special so it can be removed entirely

Reply:
Class deleted